src/lib/layouts/workflow-header.svelte

```diff
@@ -99,6 +103,7 @@
         namespace,
         query: $workflowsQuery || query,
         search: searchType,
+        page: $workflowsPage,
```

Comment:
I've opted to just keep track of the page in state rather then use `history.back()` here since the user could get to the workflows detail page from a link and we want the `Back to Workflows` button to take us to the last workflows page the user was on instead of potentially to another url entirely.